reference-apps/aaos-unbundled/mediacompose/src/main/java/com/android/designcompose/reference/mediacompose/Modules.kt

```diff
@@ -86,6 +86,8 @@ class ErrorFrameModule(
 
 @DesignModuleClass
 class BrowseSourceButtonModule(
+    @DesignProperty(node = "#media/browse/source-icon") val browseAppIcon: Bitmap?,
+    @DesignProperty(node = "#media/browse/source-name") val browseAppName: String,
```

Comment:
I made some change here and it will need to update the figma files to update the ids of browse source button to show the current browsing media app's info

Reply:
Great! I don't think we ever figured out why we couldn't browse a source different from the one currently playing. We'll need to update the node names in all the media designs. Here are the other ones:
https://www.figma.com/file/S3n4mhNgoHzNxCCHhmrVcR/Media2-V2?type=design&node-id=29%3A390&mode=design&t=kx31P8qSn329OxEa-1
https://www.figma.com/file/5n0LhOQ6wOiDxrH0YUVhJS/Media4-V2?type=design&node-id=29%3A390&mode=design&t=h6OqHIewGYZ6oOoH-1
https://www.figma.com/file/dui99iAKZ273s7RN11Z9Ak/Media5-V2?type=design&node-id=1112%3A445&mode=design&t=Z53dU1i70NBhDjsm-1
https://www.figma.com/file/2DQtQOf6U26mA8dqBie3gT/MediaNova-V2?type=design&node-id=1544%3A182&mode=design&t=1W6K3Fxs9XMNSVVN-1

Reply:
Done.